src/Migration/AbstractMigration.php

```diff
@@ -241,13 +241,17 @@ private function reset()
     private function prepare()
     {
         $queryBuilder = $this->adapter->getQueryBuilder();
+        $structure = $this->adapter->getStructure();
         $queries = [];
         foreach ($this->queriesToExecute as $queryToExecute) {
             if (!($queryToExecute instanceof MigrationTable)) {
                 $queries[] = $queryToExecute;
                 continue;
             }
+
+            $queryToExecute = $structure->prepare($queryToExecute);
```

Comment:
Maybe we should add option --strict to migrate and rollback command and validate queries only if it is used. Then we can switch it as default